user/getting-started.md

```diff
@@ -1,4 +1,35 @@
 
 # Getting Started
 
-* **TODO**
\ No newline at end of file
+Right now, 2017Q2 the easiest way to get started is to use local driver development environment: manually clone and build a container for a language driver with an SDK, then run it through Docker in order to get source -> UAST.
+
+I.e for a [Java driver](https://github.com/bblfsh/java-driver) :
+
+## Pre-requests
+ - Docker
+ - Go, for SKD build
+ - (macOS) `coreutils` and `gettext`, for SDK to work
+
+
+## Build
+```
+# build SDK
+go get -u github.com/bblfsh/sdk/...
+
+# build a driver + container
+git clone https://github.com/bblfsh/java-driver.git
+make build
+```
+
+In future, this will be replaced by `docker fetch` for the pre-build language driver containers
+
+## Run
+Get latest commit hash and run
+
+```
+cat 'native/src/main/java/bblfsh/Request.java' | docker run -it bblfsh/java-driver:dev-<commit[:6]> parse-native
```

Comment:
Feel free to send a PR so that if parse-native has an option (e.g. `--stdin`) it reads stdin instead of a file.